File: 1635-number-of-good-pairs/number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        i = 0
        gud_pairs = 0
        n = len(nums)
        while i < n:
           j = i + 1
           while j < n:
               if nums[i] == nums[j] and i < j:
                   gud_pairs += 1
               j += 1
           i += 1
        return gud_pairs

File: 1635-number-of-good-pairs/number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int i = 0;
        int gud_pairs = 0;
        int n = nums.size();
        while(i < n){
            int j = i + 1;
            while(j < n){
                if(nums[i] == nums[j]){
                    gud_pairs = gud_pairs + 1;
                }
                j++;
            }
            i++;
        }
        return gud_pairs;
    }
};